packages/build/src/commands/core_command.js

```diff
@@ -16,7 +16,7 @@ const fireCoreCommand = async function ({
   netlifyConfig,
 }) {
   try {
-    await coreCommand({
+    const commandResult = await coreCommand({
```

Comment:
[sand] Should we directly `return await coreCommand(...)`? Like this, we support core command returning other properties in the future. Also, the timers logic already adds default values in case the core commands returns `undefined`, or returns an `undefined` tags.